spec/services/distribution_itemized_breakdown_service_spec.rb

```diff
@@ -33,13 +33,11 @@
     let(:service) { described_class.new(organization: organization, distribution_ids: distribution_ids) }
 
     it "should output the expected output but in CSV format" do
-      expected_output_csv = CSV.generate do |csv|
-        csv << ["Item", "Total Distribution", "Total On Hand"]
-
-        expected_output.each do |row|
-          csv << [row[:name], row[:distributed], row[:current_onhand]]
-        end
-      end
+      expected_output_csv = <<~CSV
+        Item,Total Distribution,Total On Hand
+        Item 1,500,100
+        Item 2,200,200
```

Comment:
These are named `A Diapers` and `B Diapers`. Spec might pass with:
```suggestion
        A Diapers,500,100
        B Diapers,200,200
```